content/rs/administering/cluster-operations/updating-certificates.md

```diff
@@ -5,13 +5,13 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-Redis Enterprise Software (RS) uses self-signed certificates to encrypt
-the following traffic:
+Redis Enterprise Software (RS) uses self-signed certificates in its out of the box configuration to ensure the product is secure by default. This self-signed certificate is used to establish encryption-in-transit for the following traffic:
 
-- Management UI
-- REST API
-- Connections between clients and the database endpoint
+- RS Management User Interface (UI)
+- RS's REST API
+- Connections between clients and database endpoints
 - Synchronization between databases for ReplicaOf and CRDB
+- Discovery Services 
```

Comment:
@Paz-Yanover Do you agree that we should add this one?

Reply:
Discovery service (- sentinel) is part of the previous item of "connections between clients and database endpoint".
I think we don't have a dedicated cert for sentinel. @benamizilber - is it correct?

Reply:
@Paz-Yanover the sentinel uses the same cert as the proxy.

Reply:
@Paz-Yanover Any reason to mention "Discovery service" in that line?

Reply:
@bmansheim - no